compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1075,6 +1075,12 @@ TR::Register *OMR::X86::TreeEvaluator::integerStoreEvaluator(TR::Node *node, TR:
    if (instr && node->getOpCode().isIndirect())
       cg->setImplicitExceptionPoint(instr);
 
+   if (node->getSymbol()->isUnsafeShadowSymbol())
```

Comment:
This version only handles store, I'll update the changeset to add support for load.